Shared/Samples/Feature layer query/README.md

```diff
@@ -0,0 +1,36 @@
+# Feature layer query
```

Comment:
Hi Phil, can you help rename the sample to "Query feature table" according to the common design? There was an effort to standardize the names across platforms, and the old iOS sample name deviates from the common design. Thanks.

- the folder name
- README title
- title in metadata
- swift filename
- view type name (FeatureLayerQueryView -> QueryFeatureTableView)

Reply:
Ah, I saw there were different names in the design and the old samples; I picked the wrong one. I'll make the changes.